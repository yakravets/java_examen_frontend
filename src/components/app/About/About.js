import React from 'react';
import { PageHeader } from 'antd';

export default function About() {
    return (
      <>
        <PageHeader
          className="site-page-header"
          onBack={() => window.history.back()}
          title="About"
          subTitle="Non comercial project. With love to you."
        />
      </>);
  }