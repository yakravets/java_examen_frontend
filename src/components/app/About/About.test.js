import React from "react";

import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import About from './About';
import toJson from "enzyme-to-json";

it("renders correctly", () => {
    const tree = shallow(<About />);
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow(<About />);
});
