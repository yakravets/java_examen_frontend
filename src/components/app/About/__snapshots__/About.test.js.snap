// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <PageHeader
    className="site-page-header"
    onBack={[Function]}
    subTitle="Non comercial project. With love to you."
    title="About"
  />
</Fragment>
`;
